DIKUArcade/GUI/Window.cs

```diff
@@ -9,10 +9,14 @@
 using OpenTK.Graphics.OpenGL;
 using DIKUArcade.Input;
 
+using DIKUArcade.Events;
+
 namespace DIKUArcade.GUI {
     /// <summary>
     /// This class represents a graphical window in the DIKUArcade game engine.
     /// </summary>
+   // public class Window: IGameEventProcessor {
```

Comment:
Why retain the comment? If it is to retain a trace that perhaps `Window` should implement `IGameEvenProcessor`, then I would write this as:

```csharp
public class Window { // : IGameEventProcessor
```